docs/api/utilities.rst

```diff
@@ -44,6 +44,7 @@ Privacy-Sensitive Optax Methods
 
 .. autosummary::
     DifferentiallyPrivateAggregateState
+    dpsgd 
```

Comment:
please remove DifferentiallyPrivateAggregateState,  dpsgd and differentially_private_aggregate methods from utilities.rst so that they are only in contrib.rst